### PR TITLE
(pp) Enhancements to some Cloud M365 monitoring procedures

### PR DESCRIPTION
**Pull request by Sims24**
## Description

bye-adios

Bring some enhancements to Cloud M365 monitoring procedures

- office365-skype - Switch FR<>EN and modernize installation section
- office365-exchange - Update installation steps & troubleshooting section
- Office365 OneDrive+Skype - Update installation steps 

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [x] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

**Comment by ponchoh** [comment]
# 👋

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 1]
```diff
@@ -49,32 +51,34 @@ See link for details about metrics :
...
-referring not only to the application software, but also to the Azure AD
-registration and role in authentication/authorization "conversations" at runtime.
-(https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals)
+### Spécifiez les autorisations dont votre application a besoin pour accéder aux API de gestion Office 365
 
-### Office365 Management API authorization
-
-To collect data from Skype Online, you need to specify the following
-authorization:
+Afin de récupérer les données d'Skype Online, vous devez spécifier les
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:79]:
```suggestion
Afin de récupérer les données de Skype Online, vous devez spécifier les
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 2]
```diff
@@ -49,32 +51,34 @@ See link for details about metrics :
...
+autorisations que votre application requiert: 
+dans le Portail de gestion Azure :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:81]:
```suggestion
Dans le portail de gestion Azure :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 3]
```diff
@@ -83,49 +87,89 @@ authorization:
...
 
 ## Installation
 
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:103]:
```suggestion
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 4]
```diff
@@ -83,49 +87,89 @@ authorization:
...
+```
 
 </TabItem>
 </Tabs>
 
 ## Configuration
 
-* Log into Centreon and add a new Host through "Configuration > Hosts".
-* Apply the *Cloud-Microsoft-Office365-Skype-Api-custom* template and configure all the mandatory Macros :
+Lors de la création de votre Hôte dans Centreon, choisissez le modèle
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:170]:
```suggestion
Lors de la création de votre hôte dans Centreon, choisissez le modèle
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 5]
```diff
@@ -135,12 +179,12 @@ yum install centreon-pack-cloud-microsoft-office365-skype
 | X         | OFFICE365CLIENTSECRET | Secret-if of your registered application                                   |
 |           | OFFICE365EXTRAOPTIONS | Any extra option you may want to add to the command (eg. a --verbose flag) |
 
-The metric perfdateY will record the date the metric was collected. You can
-filter it by entering ```--filter-perfdata='^(?!.*perfdate).*$'``` into the
-*OFFICE365EXTRAOPTIONS* macro.
+La métrique *perfdate* enrengistre la date à laquelle celle-ci a été collectée.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:182]:
```suggestion
La métrique *perfdate* enregistre la date à laquelle celle-ci a été collectée. 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 6]
```diff
@@ -135,12 +179,12 @@ yum install centreon-pack-cloud-microsoft-office365-skype
...
-The metric perfdateY will record the date the metric was collected. You can
-filter it by entering ```--filter-perfdata='^(?!.*perfdate).*$'``` into the
-*OFFICE365EXTRAOPTIONS* macro.
+La métrique *perfdate* enrengistre la date à laquelle celle-ci a été collectée. 
+Vous pouvez la filter en paramétrant la macro *OFFICE365EXTRAOPTIONS* avec 
+l'option ```--filter-perfdata='^(?!.*perfdate).*$'```
 
-Once the host created, you can configure some Macros on the services to filter
-information:
+Une fois l'hôte créé, il est également possible de paramétrer un ensemble de m
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:186]:
```suggestion
Une fois l'hôte créé, il est également possible de paramétrer un ensemble de
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-azuread.md [hunk 7]
```diff
@@ -34,6 +34,19 @@ It brings the following service template:
 
 ## Prerequisites
 
+### Register an application 
+
+Refer to the documentation available [here](./cloud-microsoft-office365-management.md#prerequisites) to register your application.
+
+### Office365 Management API authorization
+
+To collect data from Onedrive Online, you need to specify the following
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-azuread.md:43]:
```suggestion
To collect data from OneDrive Online, you need to specify the following
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 8]
```diff
@@ -6,39 +6,38 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-## Vue d'ensemble
+## Overview
+Microsoft’s Office365 suite includes Skype, which is an application that
+provides video chat, voice calls and messages services between different types
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:11]:
```suggestion
provides video chat, voice calls and message services between different types
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 9]
```diff
@@ -6,39 +6,38 @@ import Tabs from '@theme/Tabs';
...
-La suite Microsoft Office 365 inclut Skype, une application offrant différents
-types de services comme la messagerie instantanée et les appels téléphoniques et
-vidéos. 
+The monitoring information of Microsoft's Office365 is available
+through the Office365 API Management.
 
-Les informations de supervision de la suite Office sont mises à disposition par
-Microsoft au travers d'une API de gestion Office 365.
+> The data provided by the Office365 Management API are not real-time.
+> They're based on a 7 days reporting period
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:18]:
```suggestion
> They're based on a 7 days reporting period.
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 10]
```diff
@@ -50,34 +49,34 @@ de Microsoft :
...
 
-Une fois l'hôte crée, les macros de services peuvent être configurées pour
-filtrer les métriques par utilisateurs ou par boites mail. Plus d'informations
-dans la section [Configuration](#Configuration).
-
 </TabItem>
 </Tabs>
 
-## Prérequis
+Once the host created, you can configure some macros on the service to filter
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:55]:
```suggestion
Once the host is created, you can configure some macros on the service to filter 
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 11]
```diff
@@ -86,49 +85,90 @@ dans le Portail de gestion Azure :
...
 
-1. Installer le Plugin sur l'ensemble des collecteurs Centreon supervisant des ressources Office 365 Skype :
+### Plugin
+
+Since Centreon 22.04, you can benefit from the 'Automatic plugin installation' feature.
+When this feature is enabled, you can skip the installation part below.
+
+You still have to manually install the plugin on the poller(s) when:
+- Automatic plugin installation is turned off
+- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:138]:
```suggestion
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet.
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 12]
```diff
@@ -138,12 +178,12 @@ certaines Macros liées à l'Hôte doivent être renseignées :
 | X         | OFFICE365CLIENTSECRET | Secret-if of your registered application                                   |
 |           | OFFICE365EXTRAOPTIONS | Any extra option you may want to add to the command (eg. a --verbose flag) |
 
-La métrique *perfdate* enrengistre la date à laquelle celle-ci a été collectée. 
-Vous pouvez la filter en paramétrant la macro *OFFICE365EXTRAOPTIONS* avec 
-l'option ```--filter-perfdata='^(?!.*perfdate).*$'```
+The metric perfdateY will record the date the metric was collected. You can
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:181]:
```suggestion
The metric perfdate will record the date the metric was collected. You can
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md [hunk 13]
```diff
@@ -138,12 +178,12 @@ certaines Macros liées à l'Hôte doivent être renseignées :
...
+filter it by entering ```--filter-perfdata='^(?!.*perfdate).*$'``` into the
+*OFFICE365EXTRAOPTIONS* macro.
 
-Une fois l'hôte créé, il est également possible de paramétrer un ensemble de m
-macros de service selon la configuration souhaitée :
+Once the host created, you can configure some Macros on the services to filter
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-skype.md:185]:
```suggestion
Once the host is created, you can configure some macros on the services to filter
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by itoussies — APPROVED** [review]

**Pull request merged**